shared/src/main/scala/amf/client/remod/AMFGraphConfiguration.scala

```diff
@@ -87,39 +130,60 @@ private[amf] class AMFGraphConfiguration(
   private[amf] def getExecutionContext: ExecutionContext    = resolvers.executionContext.executionContext
 }
 
-private[amf] object AMFGraphConfiguration {
+sealed abstract class BaseAMFConfigurationSetter(private[amf] val resolvers: AMFResolvers,
+                                                 private[amf] val errorHandlerProvider: ErrorHandlerProvider,
+                                                 private[amf] val registry: AMFRegistry,
+                                                 private[amf] val logger: AMFLogger,
+                                                 private[amf] val listeners: Set[AMFEventListener],
+                                                 private[amf] val options: AMFOptions) {
+  protected def _withParsingOptions[T](parsingOptions: ParsingOptions): T =
+    copy(options = options.copy(parsingOptions = parsingOptions)).asInstanceOf[T]
 
-  /**
-    * Predefined AMF core environment with
-    * AMF Resolvers predefined {@link amf.client.remod.amfcore.config.AMFResolvers.predefined()}
-    * Default error handler provider that will create a {@link amf.client.parse.DefaultParserErrorHandler}
-    * Empty AMF Registry: {@link amf.client.remod.amfcore.registry.AMFRegistry.empty}
-    * MutedLogger: {@link amf.client.remod.amfcore.config.MutedLogger}
-    * Without Any listener
-    */
-  def empty(): AMFGraphConfiguration = {
-    new AMFGraphConfiguration(
-        AMFResolvers.predefined(),
-        DefaultErrorHandlerProvider,
-        AMFRegistry.empty,
-        MutedLogger,
-        Set.empty,
-        AMFOptions.default()
-    )
-  }
+  protected def _withRenderOptions[T](renderOptions: RenderOptions): T =
+    copy(options = options.copy(renderOptions = renderOptions)).asInstanceOf[T]
 
-  def predefined(): AMFGraphConfiguration =
-    empty()
-      .withPlugins(List(AMFGraphParsePlugin, AMFGraphRenderPlugin))
```

Comment:
new predefined config does not set graph rendering and parsing plugins as well as basic resolution pipeline, was this change intentional?

Reply:
Claramente fue sin querer